gcloud-java-dns/src/main/java/com/google/gcloud/spi/DnsRpc.java

```diff
@@ -52,5 +58,114 @@ Integer getInt(Map<Option, ?> options) {
     }
   }
 
-  //TODO(mderka) add supported operations. Created issue #594.
+  class Tuple<X, Y> {
```

Comment:
Per your issue #615, I think in this case API can look better if you have a specific 
`ListResult<T>` class that holds Iterable<T> for values and String cursor as it seems the Tuple
is used only for listing and in a consistent way.